packages/pigment-css-react/src/styled.jsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import clsx from 'clsx';
+import isPropValid from '@emotion/is-prop-valid';
```

Comment:
What are the extra cases handled here that we need this lib ? I am mainly asking because it affects the runtime size.

Reply:
It's for filtering unwanted props to DOM for HTML tag. It is used in emotion and styled-components. If we don't add this, user will have to always specify `shouldForwardProp`:

```js
const Text = styled('div')(({ color }) => ({
  color: color,
});

<Text color="red" /> // DOM: <div color="red">
```